### ISSUE TITLE
Calculator With statements

### ISSUE THREAD

**Issue by GabrielFNBR** [orbitbreak/tkinter-calc#2]
Hi there. I need to implement in my gui a kind of calculator where the user could do how many calculations he want. I don't need a calculator layout. Actually i do prefer if it was possible to have a Blank space (a entry widget maybe?) where he could type his own expression and i need to include the possibility for if statement.

Thanks

**Comment by akshat235** [comment]
hi @GabrielFNBR do you just want to add an entry widget?

**Comment by GabrielFNBR** [comment]
> hi @GabrielFNBR do you just want to add an entry widget?

Yes, I was thinking about to implement a entry widget where the use could type any math expression, let say:

your formula here: (2 * variable_A)**1/(variable_B)
output: Result

In my code I can read the user entry as a string, but I don't know how to convert it to a function, or a expression to be evaluated and give a result.

(By the way...I didn't solve this problem since 2017...I give up this problem, but I'd thankful to implement this)

**Comment by akshat235** [comment]
Okay, I'll be happy to look into this. Please assign it to me.
